README.md

```diff
@@ -83,4 +83,152 @@ To make sure Docker automatically starts on boot, run
+
+![Sample Image 5](https://github.com/dsc-umass/lmvp/blob/f8f0b74ace33f7e4d4409bb473d3fa11d4b59b34/docs/images/SW-5.png)
+
+## How To Register A User
+
+> To be able to use the API and access all the information, a user has to be created to access the login tokens
+
+To create a user follow the following steps (Beware test Users have already been created, details below):
+
+    • Run these command in terminal
+    • export DB_PASSWORD=arbitrarypassword
+    • docker-compose run web python3 manage.py migrate
+    • docker-compose up
+    • On your web browser access the API using http://localhost:8000
+
+
+• When the API has loaded up, you would see an interphace with various function as shown in the image below
+
+![Sample Image 6](https://github.com/dsc-umass/lmvp/blob/f8f0b74ace33f7e4d4409bb473d3fa11d4b59b34/docs/images/SW-6.png)
+
+![Sample Image 7](https://github.com/dsc-umass/lmvp/blob/f8f0b74ace33f7e4d4409bb473d3fa11d4b59b34/docs/images/SW-7.png)
+
+• This shows various options and function. Click the function that states `(“/auth/register/)`
+
+• Upon click `(“/auth/register/)`, you will see an option that states try it out, this will show you an area where you will have to input some parameters (email, username and password) accordingly. 
+
+• Once this has been entered click `EXECUTE`
+
+![Sample Image 8](https://github.com/dsc-umass/lmvp/blob/f8f0b74ace33f7e4d4409bb473d3fa11d4b59b34/docs/images/SW-8.png)
+ 
+• After a few seconds you will see a Response Code 201 (CREATED) with the username and email entered, showing that the user has been successfully registered.
+
+![Sample Image 9](https://github.com/dsc-umass/lmvp/blob/f8f0b74ace33f7e4d4409bb473d3fa11d4b59b34/docs/images/SW-9.png)
+ 
+
+## How To Create Superuser
+
+In order to log into the admin interface, you have to create a superuser. To create a super user follow the following steps:
+
+    •	Run these command in terminal
+    •	$ export DB_PASSWORD=arbitrarypassword
+    •	$ docker-compose run web python3 manage.py createsuperuser
+    •	Fill in the details accordingly 
+
+## How To Log In To Admin Interface
+
+> In order to access the admin interface , you have to use your created superuser details. 
+
+In order to acces the admin interphace follow the following steps:
+
+    • $ export DB_PASSWORD=arbitrarypassword
+    • $ docker-compose run web python3 manage.py migrate
+    • $ docker-compose up
+    • Access localhost http://localhost:8000/admin from your web browser
+    • Login in with superuser details
+
+## Sample Users Created for Testing purposes
+
+## User 1
+
+    • Username: lmvpsample1
+    • Email: sample1@lmvp.com
+    • Password: lmvp_sample@1
+
+## User 2
+    • Username: lmvpsample2
+    • Email: sample2@lmvp.com
+    • Password: lmvp_sample@2

```

Comment:
The changes you make to the database are not part of the repository and just stay on your local computer. This is by design, so everyone using the app can have their own data. Therefore other people will not see the sample users you created, so I suggest removing this section

```suggestion
```